inference/infer.py

```diff
@@ -15,7 +15,7 @@
 args = TrainScriptArgsParser()
 args.load_config_file()
 
-score_cutoff = 0.001 # We won't send back any results with a score lower than this
+score_cutoff = 0.01  # We won't send back any results with a score lower than this
```

Comment:
Feels like this should be configuration

Reply:
Probably. There's a bunch of stuff in there that could probably llive in an inference configuration file yeah